run_integration.py

```diff
@@ -0,0 +1,13 @@
+import os
```

Comment:
So can this file be deleted?

Reply:
Deleted in the other PR, where markers are added.